docs/user-guide/configuring/ignoring-code.md

```diff
@@ -61,21 +60,21 @@ Please see [`.gitignore`](https://git-scm.com/docs/gitignore)'s specification fo
 In addition to any patterns in the `.eslintignore` file, ESLint always follows a couple of implicit ignore rules even if the `--no-ignore` flag is passed. The implicit rules are as follows:
 
 * `node_modules/` is ignored.
-* Dotfiles (except for `.eslintrc.*`), as well as Dotfolders and their contents, are ignored.
+* Dot-files (except for `.eslintrc.*`), as well as Dot-folders and their contents, are ignored.
 
 There are also some exceptions to these rules:
 
-* If the path to lint is a glob pattern or directory path and contains a Dotfolder, all Dotfiles and Dotfolders will be linted. This includes sub-dotfiles and sub-dotfolders that are buried deeper in the directory structure.
+* If the path to lint is a glob pattern or directory path and contains a Dot-folder, all Dot-files and Dot-folders will be linted. This includes dot-files and dot-folders that are buried deeper in the directory structure.
 
-  For example, `eslint .config/` will lint all Dotfolders and Dotfiles in the `.config` directory, including immediate children as well as children that are deeper in the directory structure.
+  For example, `eslint .config/` will lint all Dot-folders and Dot-files in the `.config` directory, including immediate children as well as children that are deeper in the directory structure.
 
 * If the path to lint is a specific file path and the `--no-ignore` flag has been passed, ESLint will lint the file regardless of the implicit ignore rules.
 
   For example, `eslint .config/my-config-file.js --no-ignore` will cause `my-config-file.js` to be linted. It should be noted that the same command without the `--no-ignore` line will not lint the `my-config-file.js` file.
 
 * Allowlist and denylist rules specified via `--ignore-pattern` or `.eslintignore` are prioritized above implicit ignore rules.
 
-  For example, in this scenario, `.build/test.js` is the desired file to allowlist. Because all Dotfolders and their children are ignored by default, `.build` must first be allowlisted so that eslint becomes aware of its children. Then, `.build/test.js` must be explicitly allowlisted, while the rest of the content is denylisted. This is done with the following `.eslintignore` file:
+  For example, in this scenario, `.build/test.js` is the desired file to allowlist. Because all Dot-folders and their children are ignored by default, `.build` must first be allowlisted so that eslint becomes aware of its children. Then, `.build/test.js` must be explicitly allowlisted, while the rest of the content is denylisted. This is done with the following `.eslintignore` file:
```

Comment:
```suggestion
  For example, in this scenario, `.build/test.js` is the desired file to allowlist. Because all dot-folders and their children are ignored by default, `.build` must first be allowlisted so that eslint becomes aware of its children. Then, `.build/test.js` must be explicitly allowlisted, while the rest of the content is denylisted. This is done with the following `.eslintignore` file:
```